docs/config.rst

```diff
@@ -36,6 +36,7 @@ The configuration options are:
 - **GHC_METADATA_CACHE_SECS**: metadata, "Capabilities Docs", cache expiry time, default 900 secs, -1 to disable
 - **GHC_REQUIRE_WEBAPP_AUTH**: require authentication (login or Basic Auth) to access GHC webapp and APIs (default: ``False``)
 - **GHC_BASIC_AUTH_DISABLED**: disable Basic Authentication to access GHC webapp and APIs (default: ``False``), see below when to set to `True`
+- **GHC_VERIFY_SSL**: disable SSL verification for Probe request calls (default: ``False``)
```

Comment:
Thanks. Two remarks:
* make text clearer: like _perform SSL verification for Probe HTTPS requests_
* Default is ``True``